xarray/tests/test_missing.py

```diff
@@ -291,7 +300,16 @@ def test_interpolate_limits():
 
 @requires_scipy
 def test_interpolate_methods():
-    for method in ["linear", "nearest", "zero", "slinear", "quadratic", "cubic"]:
+    for method in [
+        "linear",
+        "nearest",
+        "zero",
+        "slinear",
+        "quadratic",
+        "cubic",
+        "previous",
```

Comment:
we should check that the output matches that from scipy since we can't use pandas for that

Can you add that check here?